src/panels/lovelace/components/hui-generic-entity-row.ts

```diff
@@ -43,32 +45,39 @@ class HuiGenericEntityRow extends LitElement {
     }
 
     return html`
-      <state-badge
-        .stateObj="${stateObj}"
-        .overrideIcon="${this.config.icon}"
-      ></state-badge>
-      <div class="flex">
-        <div class="info">
-          ${this.config.name || computeStateName(stateObj)}
-          <div class="secondary">
-            ${!this.showSecondary
-              ? html`
-                  <slot name="secondary"></slot>
-                `
-              : this.config.secondary_info === "entity-id"
-              ? stateObj.entity_id
-              : this.config.secondary_info === "last-changed"
-              ? html`
-                  <ha-relative-time
-                    .hass="${this.hass}"
-                    .datetime="${stateObj.last_changed}"
-                  ></ha-relative-time>
-                `
-              : ""}
+      <div
+        class="host"
+        @ha-click="${this._handleTap}"
```

Comment:
I don't think that this is a good idea.

- How does this work with interactions with the state controls like turning a light on/off ? Would that still be able to do long press too ?
- What about entity rows that do not use generic-entity-row? For example, if we add the mini-media-player, this won't work?

Reply:
* in my tests I didn't have issues when using toggle rows. are you asking if you can long press the toggle?
* no, it would not. so you would prefer I add click/hold to each row instead of the base?

Reply:
@balloob any further input on what you think should happen here?